bundles/org.openhab.ui.basic/src/main/resources/snippets/main.html

```diff
@@ -68,6 +68,7 @@
 				</div>
 			</div>
 		</script>
+		<script src="https://code.iconify.design/iconify-icon/1.0.3/iconify-icon.min.js"></script>
```

Comment:
@ghys : what will happen if the user has no internet connection? Will it block the loading of the page? 
I could add a setting to enable or disable usage of iconify.

Reply:
I will add this option.

Reply:
Depends but since you're using the Iconify web component, it has a cache feature which seems to be enabled by default.
https://docs.iconify.design/iconify-icon/enable-cache.html

> By default, localStorage is enabled, sessionStorage is disabled.

This means if an icon has already been cached locally it will still work even without an internet connection. But if it isn't in the cache I don't think it will block the loading, you might just get a placeholder.

Reply:
I was thinking about the script loading. 
In fact, the best would be to test BasicUI without internet connection. But not so easy to cut internet access while keeping local network.

Reply:
Ah right, I didn't look at the change. I don't suppose everyone will be happy with having a JS file loaded from the internet in their Basic UI...

Reply:
Ok so I will add an option.

Reply:
Option now implemented. Default is to not use iconify.

Reply:
What about embedding the file locally as resource?

Reply:
Thank you @lsiepel , it works well with the file put as local resource.